tags

```diff
@@ -0,0 +1,941 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
Please feel free to add to `.gitignore`, or you may consider adding it globally to `~/.config/git/ignore`.